src/reql/mod.rs

```diff
@@ -1,2 +1,2 @@
 mod command;
-mod tree;
+pub mod tree;
```

Comment:
In this case it's strictly a necessity - the parent module needs to access this, so it must be made public.  The parent library will not expose this module to the user of this library, though - `reql` is not public: https://github.com/jimberlage/rethinkdb.rs/blob/feature/wip/src/lib.rs#L13